Makefile

```diff
@@ -1,9 +1,21 @@
-make test_backend:
-	docker-compose run --rm backend python3 manage.py test && pylint apps
+test_backend:
+	docker-compose run --rm backend python3 manage.py test
 
-make test_frontend:
+test_frontend:
 	docker-compose run --rm frontend node_modules/.bin/ng test --single-run
 
-make test_all:
-	make test_backend
-	make test_frontend
+test_all: test_backend test_frontend
+
+populate_database:
+	docker-compose run --rm backend python3 manage.py populate_database
+
+# Beware - Following commands delete ALL docker images and containers!
```

Comment:
I don't know if we need to leave this warning here - and it's a little out-of-date as it does not remove ALL images and containers